tests/test_h2o_benchmarks.py

```diff
@@ -0,0 +1,129 @@
+"""
+h2o-ai benchmark groupby part running on coiled.
+"""
+
+import uuid
+
+import dask.dataframe as dd
+import pandas as pd
+import pytest
+from coiled._beta import ClusterBeta as Cluster
+from dask.distributed import Client
+
+
+@pytest.fixture(scope="module")
+def cluster(runtime_software_env):
+    with Cluster(
+        software=runtime_software_env,
+        name="h2o-groupby-benchmark_" + str(uuid.uuid4()),
+        account="dask-engineering",
+        n_workers=10,
+    ) as cluster:
+        yield cluster
```

Comment:
I didn't see this until after I submitted https://github.com/coiled/coiled-runtime/pull/40

I recommend the changes there for a few reasons:

1.  software seems simpler than runtime_software_env, unless we think that there are going to be other kinds of software environments
2. we should specify VM types I think.  They an float otherwise and get different kinds of performance
3. I think that we should just use generic names, but I'm open to other thoughts.

However, I also see that you want a module-level cluster.  If so then yeah, this should maybe be separate.